audio/filter/af_rubberband.c

```diff
@@ -99,13 +99,23 @@ static int control(struct af_instance *af, int cmd, void *arg)
         return AF_OK;
     case AF_CONTROL_COMMAND: {
         char **args = arg;
+        char *endptr;
+        double pitch = p->pitch;
         if (!strcmp(args[0], "set-pitch")) {
-            char *endptr;
-            double pitch = strtod(args[1], &endptr);
+            pitch = strtod(args[1], &endptr);
             if (*endptr || pitch < 0.01 || pitch > 100.0)
                 return CONTROL_ERROR;
             update_pitch(af, pitch);
             return CONTROL_OK;
+        } else if (!strcmp(args[0], "multiply-pitch")) {
+            double mult = strtod(args[1], &endptr);
+            if (*endptr || mult <= 0)
+                return CONTROL_ERROR;
+            pitch = p->pitch * mult;
```

Comment:
My idea of setting pitch above was to use pitch *= mult here. If you'd prefer this way, then there's no reason to set pitch above.

Reply:
my mistake, I fixed it